pkg/sfu/receiver.go

```diff
@@ -402,6 +410,18 @@ func (w *WebRTCReceiver) downTrackLayerChange(layers []int32) {
 	for _, dt := range w.downTrackSpreader.GetDownTracks() {
 		dt.UpTrackLayersChange(layers)
 	}
+	if len(layers) > 0 && w.onMaxLayerChange != nil {
+		sort.Slice(layers, func(i, j int) bool {
```

Comment:
Don't need to sort this. This should be sorted in ascending order. So, you can take the last element as the highest available layer. Note that this does not mean the max layer has changed. We do have cases where the stream tracker will detect layer 1 stopping while layer 2 continues (or layer 1 stop may have been detected just before detecting layer 2 stop). So, wherever this is used finally may need some additional check that indeed there is a change. Or need some memory here to check for change (maybe this can be moved into `StreamTrackerManager` itself.